glusterd2/commands/volumes/volume-delete.go

```diff
@@ -45,6 +49,14 @@ func volumeDeleteHandler(w http.ResponseWriter, r *http.Request) {
 	logger := gdctx.GetReqLogger(ctx)
 	volname := mux.Vars(r)["volname"]
 
+	fnName := "?"
+	if pc, _, _, ok := runtime.Caller(0); ok {
```

Comment:
can we move this function to utils and use this function elsewhere

Reply:
Do you mean factor out this chunk to utils?

	fnName := "?"
	if pc, _, _, ok := runtime.Caller(0); ok {
		fn := runtime.FuncForPC(pc)
		fnName = filepath.Base(fn.Name())
	}

Reply:
yes, because I see we are using this function in most of the places, 
to avoid code duplication I prefer moving this function to a utils package